FE/components/organisms/VideoChat/VideoChat.tsx

```diff
@@ -0,0 +1,240 @@
+import { ReactElement, useState, useEffect } from 'react';
+import { OpenVidu, Session, StreamManager } from 'openvidu-browser';
+import { useAuthState } from '@store';
+import { Text } from '@atoms';
+import { Button } from '@molecules';
+
+import styled from 'styled-components';
+import axios from 'axios';
```

Comment:
```suggestion
import styled from 'styled-components';
import axios from 'axios';

import { useAuthState } from '@store';
import { Text } from '@atoms';
import { Button } from '@molecules';
```

컴포넌트 모듈 import 우선순위가 더 낮아서 아래에 있으면 좀 더 좋아보일 것 같아요